test/FactorialServiceSpec.js

```diff
@@ -15,10 +15,18 @@ describe('FactorialService.js', function() {
     
     it('should null if an invalid param was given', function() {
         expect(FactorialService.calculate(-1)).toBe(null);
-        expect(FactorialService.calculate("1")).toBe(null);
```

Comment:
Hi @GuilhermeSanches! 

Why did you remove this test case? It is necessary to make sure the method calculate will not support a number inside into a string.

When we have tests case we refactor the code without change the test to make sure if the business logic still valid with the new code.